kotlin/context-menu-extension/src/main/kotlin/com/example/processing/MenuActionPayload.kt

```diff
@@ -88,6 +107,6 @@ private suspend fun doCreateSubItems(client: SpaceClient, payload: MenuActionPay
 //            context.projectIdentifier,
 //            title = "${issue.title}: $it",
 //            status = status,
-//            parents = listOf(context.issueIdentifier)
+//            parents = listOf(context.issueIdentifier))
```

Comment:
A typo?

Reply:
No, it's on purpose. It belongs to `client.projects.planning.issues.createIssue(` 
I guess there was a typo in the original commented code